pages/api/groups/index.js

```diff
@@ -1,32 +1,82 @@
-import dbConnect from "../../../util/dbConnect";
+import dbConnect from "../../../utils/dbConnect";
+import mongoose from "mongoose";
+import Group from "../../../models/Group";
```

Comment:
We should start using the aliases. I'll approve and merge but create a new issue for switching files over to path alias.